src/test/java/com/google/sps/PlacesTest.java

```diff
@@ -88,9 +89,9 @@ public void getOnlyPlacesWithExpectedNumberOfRatingsAsHiddenGems() throws IOExce
 
   @Test 
   public void getRankedHiddenGemsBasedOnRatings() throws IOException {
-    try (LocalTestServerContext sc = new LocalTestServerContext(AllPlacesApiNearbySearchRequest)) {
-      Set<PlacesSearchResult> hiddenGems = Places.getAllHiddenGems(Places.fetchAllPlacesFromApi(sc.context, LOCATION));
-      List<PlacesSearchResult> rankedHiddenGems = Places.getRankedHiddenGems(hiddenGems);
+    try (LocalTestServerContext server = new LocalTestServerContext(AllPlacesApiNearbySearchRequest)) {
+      Set<HiddenGem> hiddenGems = Places.fetchHiddenGemsFromApi(server.context, Places.getAllHiddenPlaces(Places.fetchAllPlacesFromApi(server.context, LOCATION)));
```

Comment:
Nit - maybe make a new line for each call of function to make it easier to read?

Reply:
done